health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/viewVillage.js

```
@@ -160,7 +160,7 @@ const VillageView = () => {
 
 
   useEffect(() => {
-    if (processData && processData.some((instance) => instance.action === "FINALIZE_CATCHMENT_MAPPING")) {
+    if (processData && processData.some((instance) => instance.action === "APPROVE_CENSUS_DATA")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)